lang/en/docs/tutorials/python-ml/train-ml-model.md

```diff
@@ -0,0 +1,157 @@
+
+Then, when the browser's upload window appears, we navigate to where we downloaded the file in section 1, and select it
+for upload. If the upload was successful, the file will then be visible in the dropbox.
+
+Take note of the name of the file. For the purposes of this tutorial, it should be named `data_to_train_with.csv`
+
+## 3. Copy the "Python ML Train" Workflow from the Workflow Bank
+
+Next, we select the`Bank Worfklows` button in the [left sidebar](../../ui/left-sidebar.md), which brings us to
+the [Bank Workflows Page](../../workflows/bank.md). We then search for the "Python ML Train" workflow owned by the "
+Curators" account, and [copy it to our account](../../workflows/actions/copy-bank.md).
+
+A diagram and detailed description of this workflow can be found
+[here](../../software-directory/machine-learning/python-ml/components.md).
+
+## 4. Create the ML Job
+
+Next, we can create a new job by selecting the `Create Job` button in the [left sidebar](../../ui/left-sidebar.md). This
+will bring us to a new job on the [Job Designer](../../jobs-designer/overview.md) page.
+
+First, we will give the job a friendly name, such as "Python ML Tutorial" (see below). Then, we will click
+the [Actions Button](../../jobs-designer/header-menu.md#Actions) (the three vertical dots in the upper-right of the job
+designer), and choose "Select Workflow."
+
+![Job Designer with Circles](../../images/tutorials/job_designer_with_python_ml_name_and_three_dots_circled.png "Job designer page")
+
+This will bring up the [Select Workflow](../../jobs-designer/actions-header-menu/select-workflow.md) dialogue. We then
+search for "Python ML Train" and click on it to bring it into our account.
+
+## 5. Configure the ML Workflow
+
+We now have our ML workflow selected. Select the [Workflows Tab](../../jobs-designer/workflow-tab.md), and we can see
+our training workflow.
+
+We can see two [subworkflows](../../workflows/components/subworkflows.md) available: `Set Up the Job`
+and `Machine Learning`.
+
+### Specify the Training Data
+
+We will first configure the `Set Up the Job` workflow to accept our training data.
+
+Begin by selecting the "Declare Training Data" workflow unit, circled below:
+
+![ML Workflow Tab](../../images/tutorials/workflows_tab_with_ml_workflow_and_declare_training_data_circled.png "Workflow Tab")
+
+We can now see the "Declare Training Data" IO unit. Because we named our file in dropbox `data_to_train_with.csv`
+earlier, we do not need to modify this unit. In a scenario where we wanted to use a different file name for our training
+data, the [Workflow Designer](../../workflow-designer/overview.md) could be used to modify the value.
+
+![Training Data IO Unit](../../images/tutorials/training_data_io_unit.png "Training Data IO Unit")
+
+!!!warning "A Word of Caution"
+> The _only_ modifications that should be made in the `Set Up the Job` subworkflow are the filenames in the "Declare
```

Comment:
Is this the correct way to add an admonition??? https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage

Reply:
Fixed in the PR that was just merged into this one